www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb

```diff
@@ -49,6 +47,14 @@ The directory where your plan is located is known as the plan context.
 
 4. If the package builds successfully, it is placed into a `results` directory at the same level as your plan.
 
+### Managing Studio flavor to enter (Docker/Linux/Windows)
+
+Depending on the platform of your host and your Docker configuration, the behavior of `hab studio enter` may vary. Here is the default behavior listed by host platform:
+
+* **Linux** - A local chrooted Linux Studio. You can force a Docker based studio by using the `-D`` flag.
+* **Mac** - A docker container based Linux Studio
+* **Windows** - A docker container based Studio where the platform depends on the mode your docker service is running, which can be toggled between Linux Containers and Windows Containers. Make sure your docker service is running in the correct mode for the kind of studio you wish to enter. You may also use the `-w` flag to enter a local, non-containerized Windows studio.
```

Comment:
Super minor, but Docker should probably be capped in these two lines (55 and 56).